src/App.tsx

```diff
@@ -188,12 +191,20 @@ class App extends Component<void, AppState> {
     this.setState({numberOfSongs: newVal});
   }
 
+  goBack(): void {
+    this.pauseCurrentPlayback();
+    this.setState({
+      partyStarted: false,
+    })
+  }
+
   render() {
     return (
       <AppContainer className="App">
         { this.state.partyStarted ?
           <PartyTime className="app-body">
             <header>
+            { this.state.activeSong ? <ArrowBackIos onClick={() => this.goBack()} style={{ float: 'left', cursor: 'pointer', color: 'white', marginTop: "1rem", marginLeft: "1rem"}}/> : "hi"}
```

Comment:
Lol may want a better fallback than "hi"

Reply:
Lol, I have commited to this branch in like 6 mos. so I can definitely clean it up a bit before I merge into `main`. I have a merge conflict I need to resolve anyway...